File: CS-Examples/10_Conversion/HtmlFileToWord.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;

namespace HtmlFileToWord
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Create a new instance of the Document class.
			Document document = new Document();

			// Load an HTML file into the document object, with XHTML validation disabled.
			document.LoadFromFile(@"..\..\..\..\..\..\..\Data\InputHtmlFile.html", FileFormat.Html, XHTMLValidationType.None);

			// Save the document as a DOCX file named "HtmlFileToWord.docx".
			document.SaveToFile("HtmlFileToWord.docx", FileFormat.Docx);

			// Dispose the document object to release resources.
			document.Dispose();

            //Launch the file.
            WordDocViewer("HtmlFileToWord.docx");
        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/10_Conversion/HtmlFileToWord.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents

Namespace HtmlFileToWord
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new Document object
			Dim document As New Document()

			' Load the HTML file into the Document object, specifying the input file path, file format as HTML, and XHTML validation type as None
			document.LoadFromFile("..\..\..\..\..\..\..\Data\InputHtmlFile.html", FileFormat.Html, XHTMLValidationType.None)

			' Save the document to a Word file in Docx format with the specified output file name
			document.SaveToFile("HtmlFileToWord.docx", FileFormat.Docx)

			' Dispose of the Document object to release resources
			document.Dispose()


			'Launch the file.
			WordDocViewer("HtmlFileToWord.docx")
		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
